### ISSUE TITLE
Set default AuthSchemeRegistry with Jersey Client

### ISSUE THREAD

**Issue by mutkur** [jersey/jersey#3676]
Hi,
ApacheHttpComponents support configuring AuthSchemeRegistry. However Jersey doesn't seem to provide an option to set default authScheme Registry. Is there some way I can configure the same? Also does Jersey provide native support to Kerberos authentication?

Thanks in advance.

**Comment by fabriziocucci** [comment]
I guess this is more a question for the [mailing list](https://jersey.github.io/mailing.html), but have you tried already something like [this](https://stackoverflow.com/a/46328074/3302776)?

**Comment by mutkur** [comment]
Yes. This is what I have done to get around it. Can this be considered as an enhancement request?
